web-components/src/[sandbox]/examples/favorite.ts

```diff
@@ -0,0 +1,39 @@
+import "@/components/favorite/Favorite";
+import { customElement, html, LitElement, property } from "lit-element";
+
+@customElement("favorite-template-sandbox")
+export class CoachTemplateSandbox extends LitElement {
+  @property({ type: Boolean }) selected = false;
+  @property({ type: String }) value = "";
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.addEventListener("favorite-elect", this.countFavorite as EventListener);
```

Comment:
not sure if "elect" is the term we want... select? toggle?

Reply:
Changed to "favorite-toggle".